packages/askui-nodejs/src/lib/copy-example-project.ts

```diff
@@ -1,24 +1,53 @@
-import { Command } from 'commander';
+import { Command, OptionValues } from 'commander';
 import path from 'path';
 import fs from 'fs-extra';
 import { getPathToNodeModulesRoot } from '../utils/path';
+import { logger } from './logger';
 
 const createProgram = () => {
   const program = new Command('askui');
   program.usage('<command> [options]');
   return program;
 };
-function copyExampleProject() {
+
+async function replaceStringInFile(filePath: string, replace: string, replacement: string) {
+  try {
+    const data = await fs.readFile(filePath, 'utf8');
+    const result = data.replace(replace, replacement);
+    await fs.writeFile(filePath, result, 'utf8');
+  } catch (error: unknown) {
+    logger.error(`Could not replace '${replace}' with ${replacement} in file ${path}`);
+    if (error instanceof Error) {
+      logger.error(error.message);
+    }
```

Comment:
Can `error` be something else than an `Error`? If not (and I guess not), I would like a cast better as it conveys that meaning, i.e.,
```suggestion
    logger.error((error as Error).message);
```

Reply:
I read up on this and for TypeScript it is always _unknown_ (See https://stackoverflow.com/questions/42618089/how-do-you-use-typed-errors-in-async-catch)

So I rewrote it with a cast that checks for the message property and if it is there it will log the message.

Reply:
@adi-wan-askui Changed and pushed!